distribution/docs/src/main/resources/content/_tables/ContentFileSystemStorageProvider.adoc

```diff
@@ -0,0 +1,26 @@
+:title: Content File System Storage Provider
+:id: org.codice.ddf.catalog.content.impl.FileSystemStorageProvider
+:type: table
+:status: published
+:application: ${ddf-catalog}
+:summary: Content File System Storage Provider.
+
+.[[org.codice.ddf.catalog.content.impl.FileSystemStorageProvider]]Content File System Storage Provider
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Content Repository File Path
+|baseContentDirectory
+|String
+|Specifies the directory to use for the content repository. A shutdown of the server is necessary for this property to take effect. If a filepath is provided with directories that don't exist, File System Provider will attempt to create them. Out of the box (without configuration), the content repository is <INSTALL_DIR>/data/content/store.
+|
```

Comment:
❓ Should we put `<INSTALL_DIR>/data/content/store` in the `Default Value` column instead of mention that in the `Description`?

Reply:
I copy-pasted this from the metatype, and I believe we want the content of these docs to match the metatype. Do you want me to make that change in both places?

Reply:
@ricklarsen ☝️

Reply:
I think it makes sense to make that change

Reply:
Done.